exchanges/btse/btse_wrapper.go

```diff
@@ -42,7 +42,7 @@ func (b *BTSE) Run() {
 			if m.Status != "active" {
 				continue
 			}
-			currencies = append(currencies, m.Symbol)
+			currencies = append(currencies, currency.NewPairFromStrings(m.BaseCurrency, m.QuoteCurrency).String())
```

Comment:
You are creating a `currency.Pair` on this line, and immediately converting it to a string. Then on the following line with `b.UpdateCurrencies` you are converting the strings to `currency.Pair`s.
Could you change `currencies` to be `[]currency.Pair`? Then you won't need to convert back and forth

Reply:
https://api.btse.com/spot/v2/markets

You can keep this as is, as symbol contains the expected currency pair format ("BTC-USD")

Reply:
I also notice the for loop is not using an index. Please change the code to something like
```
for i := range markets {
    // ...
}
```

Reply:
This still needs a look